src/adapter/http/src/data/workspace_info_handler.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use axum::extract::Extension;
+use axum::response::Json;
+use dill::Catalog;
+use http_common::*;
+use kamu_core::DatasetRepository;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug, serde::Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct WorkspaceInfoResponse {
```

Comment:
Let's call this `NodeInfo`. The definition of "workspace" as I see it is a directory on a local file system that stores datasets and other ODF objects managed by an ODF node (such as `kamu-cli`).